src/libs/actions/Card.ts

```diff
@@ -381,6 +381,24 @@ function startIssueNewCardFlow(policyID: string) {
     API.read(READ_COMMANDS.START_ISSUE_NEW_CARD_FLOW, parameters);
 }
 
+function issueExpensifyCard(policyID: string, feedCountry: string, {assigneeEmail, limit, limitType, cardTitle, cardType}: IssueNewCardData) {
+    const parameters = {
+        policyID,
+        assigneeEmail,
+        limit,
+        limitType,
+        cardTitle,
+    };
+
+    if (cardType === CONST.EXPENSIFY_CARD.CARD_TYPE.PHYSICAL) {
+        API.write(WRITE_COMMANDS.CREATE_EXPENSIFY_CARD, {...parameters, feedCountry});
```

Comment:
@koko57 Just confirm: We only add `feedCountry` param in creating a physical card

Reply:
yes, it was discussed with Maria (I cannot find where exactly though), @MariaHCD could you confirm please?